python/ql/src/Security/CWE-327/Ssl.qll

```diff
@@ -0,0 +1,186 @@
+import python
+import semmle.python.ApiGraphs
+import semmle.python.dataflow.new.internal.Attributes as Attributes
```

Comment:
Don't we expose attributes publicly? :confused: oh, you're not importing DataFlow ... I think you should import DataFlow instead, and then use `DataFlow::AttrWrite` instead of `Attributes::AttrWrite`

Reply:
Indeed, I did not even need a new import.